ci_setup.sh

```diff
@@ -26,8 +26,8 @@ echo "running ./adx build"
 adx build
 echo "running ./adx up"
 adx up
-echo "tunning ./adx/dbsetup"
-adx dbsetup
+echo "tunning ./adx testsetup"
+adx testsetup
```

Comment:
adx testsetup is already run i :36
I think it should go like this:
adx build
adx up
adx testsetup
adx dc ps (it's an alias for docker-compose ps)

So removed steps:
adx dbsetup
adx restart ckan

Reply:
done